openai.chat/client.bal

```diff
@@ -59,6 +59,7 @@ public isolated client class Client {
     # Creates a completion for the chat message
     #
     # + return - OK 
+    @display {label: "Create Chat Completion"}
```

Comment:
Shouldn't we modify the client level annotation as well, to fix the connector name shown in choreo marketplace?